test/server/grid.js

```diff
@@ -198,30 +199,30 @@ describe('Grid', function () {
             });
         });
 
-        describe('constructFrameList', function () {
+        describe('constructFrameList TODO : Moove to Player tests', function () {
```

Comment:
Move the TODO out of the describe, maybe a comment is enough

Reply:
and please Move not Moove

Reply:
done